requirements.txt

```diff
@@ -3,10 +3,11 @@ v3io~=0.5.14
 # exclude pandas 1.5.0 due to https://github.com/pandas-dev/pandas/issues/48767
 # and 1.5.* due to https://github.com/pandas-dev/pandas/issues/49203
 pandas>=1, !=1.5.*, <3
-numpy>=1.16.5,<1.23
+# <=1.26.2 is just a safeguard - no tests performed with numpy higher than 1.26.2
+numpy<=1.26.2
 # <15 is just a safeguard - no tests performed with pyarrow higher than 14
 pyarrow>=1,<15
 v3io-frames~=0.10.3
 v3iofs~=0.1.17
 xxhash>=1
-nuclio-sdk>=0.5.3
\ No newline at end of file
+nuclio-sdk>=0.5.3
```

Comment:
What's the change?

Reply:
Idk and I can not revert it, maybe it's just a GitHub mistake.

install requirements working and install the required package, so it does not seems as a problem...

Reply:
it's a formatting change - adding newline char at the end of file